src/App.js

```diff
@@ -1,51 +1,49 @@
 import './App.css';
 import axios from 'axios';
```

Comment:
Do you we will still need axios package ?

Reply:
No need, removed.